client/src/components/Home/AboutUs/Notice/Notice.css

```diff
@@ -0,0 +1,105 @@
+.notice-section .notice-title {
+    color: #1EABD6;
+    font-size: 4vh;
+    margin-left: 45vw;
+}
+
+.notice-section .notice-content .title {
+    color: white;
+    font-size: 3vh;
+    margin-left: 45vw;
+    margin-top: 3vh;
+}
+
+.notice-section .notice-content .notice-list {
+    color: white;
+    font-size: 3vh;
+    margin-left: 45vw;
+    margin-bottom: 15vh;
+}
+
+.notice-section .planet {
+    height: 0;
+}
+
+.notice-section .planet-img {
+    width: 55vw;
+    height: auto;
+    position: relative;
+    left: 0;
+    pointer-events: none;
+    bottom: 68vh;
+    z-index: 100;
+    animation: myOrbit 4s linear infinite;
+}
+
+.notice-section .mobile-planet {
+    height: 0;
+    opacity: 0;
+    pointer-events: none;
+}
+
+.notice-section .union {
+    height: 0;
+    pointer-events: none;
+}
+
+.notice-section .union-img {
+    height: 200vh;
+    width: auto;
+    position: relative;
+    left: 37vw;
+    bottom: 55vh;
```

Comment:
You can limit the height of union.svg to the about us section only for now so that it doesn't overlap with other components and whenever the community section will be extended as per the designs, height of the union.svg be configured according to it.

Reply:
Here's my though:
After this, we'll immediately start working on remaking community section of the home page which is as follows: 
![image](https://user-images.githubusercontent.com/62606998/93751826-547cc100-fc1b-11ea-8d3e-4d0cfc69fa40.png)

I've extended that line intentionally to make further development a bit easier.
But I may just remove it in this. Its your call @tusharjain0022 and @ankiiitraj.

Reply:
I think then it's absolutely ok...   LGTM!! 👍